docs/Python-2-3-compatibility.rst

```diff
@@ -0,0 +1,192 @@
+.. _py2_py3_compatibility:
+
+Compatibility with Python 2 and Python 3
+========================================
+
+Starting with EasyBuild v4.0, the EasyBuild framework and easyblocks are compatible with both Python versions 2 and 3.
+More specifically, the following Python versions are currently supported:
+
+* Python 2.6.x
+* Python 2.7.x
+* Python 3.5.x
+* Python 3.6.x
+* Python 3.7.x
+
+To facilitate this, the ``easybuild.tools.py2vs3`` package was introduced in EasyBuild v4.0.
+When importing a function from this package, you will automagically get a version of the function that
```

Comment:
hmmm "automatically" is probably better

Reply:
It's a bit like magic though... Isn't it?